### ISSUE TITLE
[Accessibility] Application menu control lacks expanded/collapsed state information

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227255]
**Issue**: There is a menu bar item (harmburger option) on top left of the VSCode editor which lacks state information such as expanded/collapsed. 

**Impact**: Screen reader users will be unable to determine that these controls are expand and collapse content. They will also have difficulty determining whether a control is currently expanded or collapsed.

**Recommendation**: 
For menu controls, the following information is expected:
- When the content area is visible, the button must have aria-expanded="true". Otherwise, the button must have aria-expanded="false".

Screenshot of issue:

<img width="54" alt="Screenshot 2024-09-02 at 10 04 43 PM" src="https://github.com/user-attachments/assets/488e1323-9a44-49e9-bb6f-593b9e765cd6">

**Comment by meganrogge** [comment]
On insider's, when I focus each activity bar item on macOS, I hear expanded or collapsed as expected.
